ironfish/src/testUtilities/types.ts

```diff
@@ -0,0 +1,4 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
```

Comment:
This type is used for test fixtures so that we can guarantee the spendingKey is present on the given account.